Initial solution background: TOC and ADRs (#3)
stitakis/ArchElekt: Solution Background/ADRs/001 Allow experts to maintain their own skills.md

# 1. Allow experts to maintain their own skills

Date: 2021-04-28

## Status

Accepted

## Context

> Customers reported that sometimes wrong expert shows up to the appointment to fix something they know nothing about. Sometimes no expert shows up at all. We assume that one of the reasons for this might be an expert that doesn't feel like being the right person for a specific issue.

Reviewer: ```suggestion
Customers reported that sometimes wrong expert shows up to the appointment to fix something they know nothing about. Sometimes no expert shows up at all. We assume that one of the reasons for this might be the expert does not have the right skill set because those are maintained by the manager and might be outdated.
```

## Decision

> To help to improve the situation, we think that experts should be able to maintain their own skills. This will help to:

Reviewer: ```suggestion
To improve the situation, experts should be able to maintain their own skills. This will help to:
```

- relief the administrator from this activity
- help the system to match tickets with better accuracy to available skillsets
- help the system to find an expert and therefore reduce the perception of lost tickets

> A possible solution is a new section in the expert's mobile app for editing/maintaining the own profile including the skills to offer.

Reviewer: ```suggestion
A possible solution is a new section in the expert's mobile app for editing/maintaining their own profile including skills.
```

## Consequences

> Allowing experts to maintain their own skills could lead to a misuse of this function. Some experts could introduce more skills than they actually have to gather more jobs. While the former situation limits the skillset entry to the administrator, who acts as a filter, it also builds a bottleneck and doesn't guarantee the accuracy of the experts skillset introduced by the administrator.

Reviewer: More likely the opposite. Experts might add less skills that they actually have to have less work :-D

> We believe that by allowing experts to maintain their own skillset will outweigh this risks and will help overall to improve the situation.

Reviewer 2: gramatical correction: `these risks` instead of `this risks`
